### PR TITLE
[#4212] Update installer scripts for IS

### PR DESCRIPTION
**Pull request by JuanCarlos-Arce**
Update cmake scripts to generate the installer for IS

### CODE REVIEW

**Review by richiware — DISMISSED** [review]

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/24/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/36/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/26/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/38/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/27/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/39/)

File: CMakeLists.txt [hunk 1]
```diff
@@ -211,18 +198,19 @@ elseif(NOT EPROSIMA_INSTALLER)
...
-    option(COMPILE_EXAMPLES "Build example" OFF)
-    if(COMPILE_EXAMPLES)
-        if(NOT CMAKE_BUILD_TYPE)
-            set(CMAKE_BUILD_TYPE "Debug" CACHE STRING "Build type" FORCE)
-        endif()
-        add_subdirectory(examples)
+# Examples
+option(COMPILE_EXAMPLES "Build example" OFF)
+if(COMPILE_EXAMPLES)
+    if(NOT CMAKE_BUILD_TYPE)
```
Comment (richiware) [inline CMakeLists.txt]: I think is better not to force to compile examples only in Debug. Leave this decision to the user. Remove this part.

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/28/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/40/)

**Review by richiware — CHANGES_REQUESTED** [review]

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/29/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/41/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/31/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/43/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/32/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/44/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/33/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/45/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/34/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/46/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/35/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/47/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/36/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/48/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/37/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/49/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/38/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/50/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/41/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/51/)

File: CMakeLists.txt (hunk 1 above)
Comment (JuanCarlos-Arce) [reply CMakeLists.txt]: Changed

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/46/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/56/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Linux/47/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Integration-Service%20Manual%20Windows/57/)

**Comment by JuanCarlos-Arce** [comment]
We have fixed an issue to generate the Linux installer

**Review by richiware — APPROVED** [review]

**Pull request merged**